File: 3110-score-of-a-string/3110-score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int sum=0;
        for (int i=0; i<s.size()-1; i++){
            int currChar= s[i];
            int nextChar= s[i+1];
            int diff= currChar-nextChar;
            sum= sum+abs(diff);
        }
        return sum;
    }
};

File: 3110-score-of-a-string/3110-score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        sum=0
        for i in range(len(s)-1):
            currChar= s[i]
            nextChar= s[i+1]
            diff= ord(currChar)-ord(nextChar)
            sum+= abs(diff)
        return sum